plugin/pkg/admission/namespace/autoprovision/admission.go

```diff
@@ -59,7 +60,7 @@ func (p *provision) Admit(a admission.Attributes) (err error) {
 		},
 		Status: api.NamespaceStatus{},
 	}
-	_, exists, err := p.store.Get(namespace)
+	_, exists, err := p.informer.GetStore().Get(namespace)
```

Comment:
doing this suggests that their lister is incomplete, let's get a followup to fix it.